### PR TITLE
US-550022: Release notes updated and version changed to 1.0.1

### PR DESCRIPTION
**Pull request by vishalkumar-barnwal**
US-550022: Release notes updated

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
     <groupId>com.github.cafapi.util.flywayinstaller</groupId>
     <artifactId>util-flywayinstaller-aggregator</artifactId>
-    <version>1.1.0-SNAPSHOT</version>
+    <version>1.0.1-SNAPSHOT</version>
```
Comment (frankmccarry) [inline pom.xml:32]: Hi Vishal,
you're going backwards in version numbering here. Look at US 551015 for directions on how to update version numbers.
Looking at the history I do not see recent changes, can this not just be released?

**Review by frankmccarry — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (michael-bryson) [reply pom.xml:32]: @frankmccarry I think 1.0.1 is the correct version to move to for this release, the update of the psql driver would not constitute a new feature so the next patch version from the last release looks correct to me.

File: pom.xml [hunk 2]
```diff
@@ -116,7 +116,7 @@
             <dependency>
                 <groupId>org.postgresql</groupId>
                 <artifactId>postgresql</artifactId>
-                <version>42.3.3</version>
+                <version>42.5.0</version>
```
Comment (michael-bryson) [inline pom.xml]: Could you merge the main branch into this branch so this does not show up as a change made by this PR?

Comment (vishalkumar-barnwal) [reply pom.xml]: Merged

File: release-notes-1.0.1.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#### Version Number
+${version-number}
+
+#### New Features
+- None
+
+#### Patch Fixes Included
+- PostgreSQL JDBC driver version upgraded
```
Comment (andyreidz) [inline release-notes-1.0.1.md]: I made a comment on the commit, remaking it here in case you missed it. 
Should you have the Octane ticket number here?

Comment (vishalkumar-barnwal) [reply release-notes-1.0.1.md]: I have included the ticket number here.

**Review by frankmccarry — APPROVED** [review]

**Pull request merged**